operator/src/main/java/oracle/kubernetes/operator/EventConstants.java

```diff
@@ -61,14 +67,15 @@
   String REPLICAS_TOO_HIGH_ERROR = "Replicas too high";
   String INTERNAL_ERROR = "Internal error";
   String ABORTED_ERROR = "Domain processing is aborted";
-  String ABORTED_ERROR_SUGGESTION = "The domain will not be retried unless the problem is corrected.";
+  String ABORTED_ERROR_SUGGESTION = "The reported problem should be corrected, and the domain will not be retried "
+      + "until the domain resource is updated.";
   String WILL_NOT_RETRY = ABORTED_ERROR_SUGGESTION;
   String WILL_RETRY = " Will retry.";
   String DOMAIN_INVALID_ERROR_SUGGESTION = "Update the domain resource to correct the validation error.";
   String TOPOLOGY_MISMATCH_ERROR_SUGGESTION
       = "Update the domain resource or change the WebLogic domain configuration to correct the error.";
   String KUBERNETES_ERROR_SUGGESTION = "";
   String SERVER_POD_ERROR_SUGGESTION = "";
-  String REPLICAS_TOO_HIGH_ERROR_SUGGESTION
-      = "Lower replicas in domain resource or increase MaxDynamicClusterSize in WebLogic domain configuration";
+  String REPLICAS_TOO_HIGH_ERROR_SUGGESTION = "Lower replicas in the domain resource, or increase DynamicClusterSize "
```

Comment:
Oops, it looks like we both forgot about configured clusters.  So we either we make it longer, something like:

Lower replicas in the domain resource, or increase DynamicClusterSize and MaxDynamicClusterSize in the WebLogic domain configuration for the cluster (in the dynamic cluster case), or configure additional WebLogic servers in the WebLogic domain configuration for the cluster (in the configured cluster case).

Or we make it shorter and assume that the end user will be able to figure it out:

Lower replicas in the domain resource, or increase the number of WebLogic servers in the WebLogic domain configuration for the cluster.

I personally favor the second (shorter) choice.

Reply:
fixed. thanks